spec/grammar-spec.coffee

```diff
@@ -145,3 +185,122 @@ describe "less grammar", ->
     expect(tokens[9]).toEqual value: ";", scopes: ['source.css.less', 'meta.property-list.css', 'meta.property-value.css', 'punctuation.terminator.rule.css']
     expect(tokens[10]).toEqual value: " ", scopes: ['source.css.less', 'meta.property-list.css']
     expect(tokens[11]).toEqual value: "}", scopes: ['source.css.less', 'meta.property-list.css', 'punctuation.section.property-list.end.css']
+
+  it 'parses variable interpolation in selectors', ->
+    {tokens} = grammar.tokenizeLine '.@{selector} { color: #0ee; }'
```

Comment:
It actually isn't really, it wasn't being tokenised at all. In 42c4632 I changed it so the variable interpolation is part of the `attribute-name.class` and `attribute-name.id` scope, which also means the `.` is correctly tokenised. Specs updated to match.